Dockerfile

```diff
@@ -1,5 +1,22 @@
 # Container image that runs your code
-FROM alpine:3.10
+FROM golang:1.14-alpine3.11 as builder
+
+ENV GO111MODULE on
+RUN go get -u github.com/google/go-jsonnet/cmd/jsonnetfmt@master
+
+
+FROM alpine:3.11 as runner
+
+COPY --from=builder /go/bin/jsonnetfmt /bin/jsonnetfmt
+
+LABEL repository="https://github.com/kakkoyun/action-jsonnetfmt"
+LABEL homepage="https://github.com/kakkoyun/action-jsonnetfmt"
+LABEL maintainer="Kemal Akkoyun <kakkoyun@gmail.com>"
+
+LABEL "com.github.actions.name"="jsonnetfmt Action"
+LABEL "com.github.actions.description"="Run jsonnetfmt to check if jsonnet files are formatted in current workspace"
+LABEL "com.github.actions.icon"="jsonnet"
```

Comment:
I'm not sure what this is YOLO

Reply:
This is same as 
```yml
name: 'Hello World'
description: 'Greet someone and record the time'
```

You're using `action.yml` no need to ass these labels.

Reply:
What about `actions.icon`?

Reply:
You should add it to `action.yml`